packages/amplify-cli/src/extensions/amplify-helpers/trigger-flow.js

```diff
@@ -53,7 +52,7 @@ const addTrigger = async triggerOptions => {
     throw new Error('Function plugin not installed in the CLI. You need to install it to use this feature.');
   }
 
-  await add(context, 'awscloudformation', FunctionServiceName.LambdaFunction, {
+  await add(context, 'awscloudformation', 'LambdaFunction', {
```

Comment:
I think this needs to be the enum

Reply:
You're correct, thanks for catching this